packages/jupyter-ai/jupyter_ai/personas/persona_manager.py

```diff
@@ -135,6 +139,10 @@ def _init_persona_classes(self) -> None:
                 "ERROR: Jupyter AI has no AI personas available. "
                 + "Please verify your server configuration and open a new issue on our GitHub repo if this warning persists."
             )
+
+        # TODO: check whether in the end we need a full class for this, or if we can just use a simple function that returns a list of persona classes from the local filesystem.
```

Comment:
Yeah, my sense so far is that a simple function will be simpler and cleaner. The only thing that I could imagine needing state for is for caching, but that can be done with a decorator.